tests/scheduler/test_scheduler.py

```diff
@@ -0,0 +1,396 @@
+#
```

Comment:
move this file to singlecard folder, or update https://github.com/vllm-project/vllm-ascend/blob/main/.github/workflows/vllm_ascend_test.yaml#L124 to enable this test